docker/mjml/Dockerfile

```
@@ -0,0 +1,8 @@
+
+FROM demisto/python3:3.12.11.4508456
```

```suggestion
FROM demisto/python3:3.12.11.4819260
```

Hi @schroeder247, please use the latest Python3 tag.